addons/editor-calculator/userscript.js

```diff
@@ -0,0 +1,101 @@
+export default async function ({ addon, global, console, msg }) {
+
+  var windowCalculator = {closed:true};

```

Comment:
Don't use `var`. Use `let` or `const` instead. We try to avoid writing to global.